runWebpack.sh

```diff
@@ -108,6 +108,7 @@ gitops_config() {
         cp $copySource/*.* $copyDest/
       else
         echo "INFO: gitops_config - no $copySource in gitops"
+        rm $copyDest
```

Comment:
if file is not found it removes any copy that could already exist

Reply:
if `copyDest` doesn't exists this will throw a bunch of errors?

Reply:
added flag to not throw errors if file does not exist